### PR TITLE
Bump indexer and core routes versions

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

**Review by Thoralf-M — DISMISSED** [review]

**Review by kwek20 — DISMISSED** [review]

**Review by Alex6323 — DISMISSED** [review]

File: .github/workflows/private-tangle-tests.yml [hunk 1]
```diff
@@ -66,7 +66,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: nextest
-          args: run --tests --all-features --no-fail-fast --run-ignored ignored-only --cargo-profile ci --retries 2
+          # args: run --tests --all-features --no-fail-fast --run-ignored ignored-only --cargo-profile ci --retries 2
```
Comment (Thoralf-M) [inline .github/workflows/private-tangle-tests.yml:69]: :trollface:

Comment (Thoralf-M) [reply .github/workflows/private-tangle-tests.yml:69]: @thibault-martinez also need to disable the wasm tests

Comment (thibault-martinez) [reply .github/workflows/private-tangle-tests.yml:69]: Why? They call nodes?

Comment (Thoralf-M) [reply .github/workflows/private-tangle-tests.yml:69]: Yes https://github.com/iotaledger/iota-sdk/actions/runs/5324418238/jobs/9643688508

Comment (thibault-martinez) [reply .github/workflows/private-tangle-tests.yml:69]: But nodejs too then?

Comment (thibault-martinez) [reply .github/workflows/private-tangle-tests.yml:69]: Looks fine

**Review by kwek20 — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Review by qrayven — APPROVED** [review]

**Pull request merged**